x-pack/plugins/gis/public/components/layer_panel/_layer_panel.scss

```diff
@@ -23,6 +23,16 @@
   box-shadow: 0 $euiSize $euiSize (-$euiSize / 2) $euiColorLightestShade;
 }
 
+.gisLayerPanel__sourceDetails {
+  margin-left: $euiSizeXL;
+}
+
+.gisLayerDetails {
+  p {
```

Comment:
Naked selectors are frowned upon in our linter. You'll want to add a class to the actual `<p>` tags.